### PR TITLE
[Fix][API] Fix that the response body does not return the token attribute after successful login

### PR DESCRIPTION
**Pull request by zhuangchong**
Fix that the response body does not return the token attribute after successful login

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-ui/.env.development [hunk 1]
```diff
@@ -15,4 +15,4 @@
 
 NODE_ENV=development
 
-VITE_APP_DEV_WEB_URL=''
+VITE_APP_DEV_WEB_URL='http://127.0.0.1:8080'
```
Comment (EricJoy2048) [inline seatunnel-ui/.env.development:18]: How about change the port? `8080` is  too common and easy to conflict.

File: seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/AuthController.java [hunk 2]
```diff
@@ -41,8 +41,8 @@ public class AuthController {
     @GetMapping("/userRole")
     @ApiOperation(value = "check relation between user and role", httpMethod = "GET")
     @ApiImplicitParams({
-            @ApiImplicitParam(name = "username", value = "user name", dataType = "String"),
-            @ApiImplicitParam(name = "roleName", value = "role name", dataType = "String"),
+        @ApiImplicitParam(name = "username", value = "user name", dataType = "String"),
```
Comment (EricJoy2048) [inline seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/AuthController.java:44]: revert?

File: seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/ScriptController.java [hunk 3]
```diff
@@ -120,7 +120,7 @@ public Result<Void> updateScriptParam(@ApiParam(value = "script id", required =
     @GetMapping("/{scriptId}/param")
     @ApiOperation(value = "fetch script param", httpMethod = "GET")
     @ApiImplicitParams({
-            @ApiImplicitParam(name = "id", value = "script id", dataType = "Integer"),
+        @ApiImplicitParam(name = "id", value = "script id", dataType = "Integer"),
```
Comment (EricJoy2048) [inline seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/ScriptController.java:123]: revert?

File: seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/AuthController.java (hunk 2 above)
Comment (zhuangchong) [reply seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/AuthController.java:44]: This is to fix the code format problem reported by `checkstyle`.

File: seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/ScriptController.java (hunk 3 above)
Comment (zhuangchong) [reply seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/ScriptController.java:123]: This is to fix the code format problem reported by `checkstyle`.

image

File: seatunnel-ui/.env.development (hunk 1 above)
Comment (zhuangchong) [reply seatunnel-ui/.env.development]: done.

**Review by songjianet — APPROVED** [review]

**Review by dijiekstra — APPROVED** [review]
LGTM

**Pull request merged**